### PR TITLE
You are so insensitive

### PR DESCRIPTION
**Pull request by nahiluhmot**
**BUG FIX** [Trello](https://trello.com/c/LpA7BwgX/32-username-email-shouldn-t-be-case-sensitive)

@Daniel0524 @tlunter 

Ignore email casing and add a one off to change all existing emails to lower case.

### CODE REVIEW

File: app/auth/services/users.rb [hunk 1]
```diff
@@ -25,13 +25,13 @@ def read(id)
 
       def update(id, hash)
         user = get_user(id)
-        email = hash['email']
+        email = hash['email'].try(:downcase)
```
Comment (tlunter) [inline app/auth/services/users.rb]: The email will still be set with the non-downcased version.

Comment (nahiluhmot) [inline app/auth/services/users.rb]: How so?

Comment (tlunter) [inline app/auth/services/users.rb]: https://github.com/cappity-cappity-capstone/auth/pull/5/files#diff-8b9df632b2201c266696324683edcfdeR38 you just pass in hash. This does not change hash. Emails will still get through that aren't downcased.

Comment (nahiluhmot) [inline app/auth/services/users.rb]: Lol didn't think I was directly passing the hash.

Comment (nahiluhmot) [inline app/auth/services/users.rb]: Lol didn't think I was directly passing the hash.

**Pull request merged**